src/main/resources/templates/db_scripts/schema.json

```
@@ -1035,6 +1035,11 @@
       "run": "after",
       "snippetPath": "inventory-hierarchy/addEffectiveHoldingsToItemsAndHoldingsView.sql",
       "fromModuleVersion": "20.2.0"
+	},
+	{
+      "run": "after",
+      "snippetPath": "addInstanceFormatsAudioBelt.sql",
+      "fromModuleVersion": "20.2.1"
```

The next planned release for mod-inventory-storage is 20.3.0, which means I think this should be 20.3.0

```suggestion
      "fromModuleVersion": "20.3.0"
```

---

okay